tools/make_hgrid/make_hgrid.c

```diff
@@ -1057,63 +1155,79 @@ int main(int argc, char* argv[])
       mpp_put_var_value_block(fid, id_tile, start, nwrite, tilename );
 
       if(out_halo ==0) {
+	if (verbose) printf("[INFO] START NC XARRAY write out_halo=0 tile number = n: %d offset = pos_c: %d\n", n, pos_c);
+	if (verbose) printf("[INFO] XARRAY: n: %d x[0]: %f x[1]: %f x[2]: %f x[3]: %f x[4]: %f x[5]: %f x[10]: %f\n", n, x[pos_c], x[pos_c+1], x[pos_c+2], x[pos_c+3], x[pos_c+4], x[pos_c+5], x[pos_c+10]);
+	if (verbose && n > 0) printf("[INFO] XARRAY: n: %d x[0]: %f x[-1]: %f x[-2]: %f x[-3]: %f x[-4]: %f x[-5]: %f x[-10]: %f\n", n, x[pos_c], x[pos_c-1], x[pos_c-2], x[pos_c-3], x[pos_c-4], x[pos_c-5], x[pos_c-10]);
```

Comment:
stderr?  Also, why not just one `if (verbose)` block, with a nested `if (n > 0)`?

Note, there are other `printf("[INFO] ...` that should be corrected.  I'm not going to continue marking them.  I  feel diagnostics, notices, warnings, etc. should be sent to stderr.

Reply:
I've gone back and adjusted these `printf("[INFO] ...` statements to be piped to stderr. I've also condensed consecutive `if (verbose)` statements into blocks.